### ISSUE TITLE
Focus search on unlocking database

### ISSUE THREAD

**Issue by rugk** [keepassxreboot/keepassxc#2279]
(seen this behavior in the "original" KeePass)

## Expected Behavior

Search should be focused.

## Current Behavior

Something else is focused, I guess it's the first entry in the database.

## Possible Solution

manually set focus

## Steps to Reproduce (for bugs)

1. Unlock database

## Context

Use case: Quick search after unlocking database, usability.

**Comment by droidmonkey** [comment]
This was literally just reported and there is a fix already! #1797

**Comment by droidmonkey** [comment]
Pressing CTRL+F focuses search. There are probably 50% of people who want an entry focused and 50% who want search focused.

**Comment by rugk** [comment]
Hmm, is a setting really needed? I mean I do not see the use case for having an entry selected. I'll guess nobody will use the arrow keys for navigating to an entry after start.
I just saw this behaviour in KeePass and liked it quite much.

**Comment by droidmonkey** [comment]
Actually if you press down arrow in the search box it will select the first entry anyway, so we can easily change focus to search box on unlock without losing anything.

**Comment by droidmonkey** [comment]
I would just like to point out that this is already the behavior! See here: https://github.com/keepassxreboot/keepassxc/blob/master/src/gui/SearchWidget.cpp#L123

**Comment by rugk** [comment]
Hmm, when I tested it in Fedora 28 it did not work, but maybe I'll need to test it again.
